components/EnumNode.tsx

```diff
@@ -1,45 +1,73 @@
 import cc from "classcat";
-import React from "react";
+import React, { useState } from "react";
 import { Handle, Position } from "react-flow-renderer";
 
 import styles from "./Node.module.scss";
 
 import { EnumNodeData } from "~/util/types";
 
-const EnumNode = ({ data }: EnumNodeProps) => (
-  <table
-    className="font-sans bg-white border-2 border-separate border-black rounded-lg"
-    style={{ minWidth: 200, maxWidth: 500, borderSpacing: 0 }}
-  >
-    <thead title={data.documentation}>
-      <tr>
-        <th
-          className="p-2 font-extrabold border-b-2 border-black bg-emerald-200 rounded-t-md"
-          colSpan={1}
-        >
-          {data.name}
-          {!!data.dbName && (
-            <span className="font-mono font-normal">&nbsp;({data.dbName})</span>
-          )}
-        </th>
-      </tr>
-    </thead>
-    <tbody>
-      {data.values.map((val) => (
-        <tr key={val} className={styles.row}>
-          <td className="p-2 font-mono border-t-2 border-gray-300">{val}</td>
+const MAX_VALUES = 12;
+
+const EnumNode = ({ data }: EnumNodeProps) => {
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <table
+      className="font-sans bg-white border-2 border-separate border-black rounded-lg"
+      style={{ minWidth: 200, maxWidth: 500, borderSpacing: 0 }}
+    >
+      <thead title={data.documentation}>
+        <tr>
+          <th
+            className="p-2 font-extrabold border-b-2 border-black bg-emerald-200 rounded-t-md"
+            colSpan={1}
+          >
+            {data.name}
+            {!!data.dbName && (
+              <span className="font-mono font-normal">
+                &nbsp;({data.dbName})
+              </span>
+            )}
+          </th>
         </tr>
-      ))}
-    </tbody>
-
-    <Handle
-      className={cc([styles.handle, styles.bottom])}
-      type="source"
-      position={Position.Bottom}
-      isConnectable={false}
-    />
-  </table>
-);
+      </thead>
+      <tbody
+        className={`flex flex-col overflow-hidden ${
+          data.values.length > MAX_VALUES && !expanded && "max-h-[500px]"
+        }`}
```

Comment:
Just to keep it consistent with the rest of what I have already.
```suggestion
        className={cc([
          "flex",
          "flex-col",
          "overflow-hidden",
          { "max-h-[500px]": !expanded && data.values.length > MAX_VALUES },
        ])}
```